datafusion/functions/src/string/ltrim.rs

```diff
@@ -81,7 +81,11 @@ impl ScalarUDFImpl for LtrimFunc {
     }
 
     fn return_type(&self, arg_types: &[DataType]) -> Result<DataType> {
-        utf8_to_str_type(&arg_types[0], "ltrim")
+        if arg_types[0] == DataType::Utf8View {
+            Ok(DataType::Utf8View)
```

Comment:
Can we possibly add a .slt test to cover this (showing that the output type is now a view and some basic end to end tests (if not already done)?) 

Perhaps in https://github.com/apache/datafusion/blob/main/datafusion/sqllogictest/test_files/string_view.slt ?

Reply:
Ok, I will fixing tests and benchmarks now.

Reply:
Fixed test, and introduced a benchamrk in #12513 .
https://github.com/apache/datafusion/pull/12395#issuecomment-2356681769 shows number about improvement.